Adding more modern real world project examples.

okrasz opened the pull request:
As we discussed on GWT.create - I've added some more modern projects using GWT. For Google I changed it from Moderator to Inbox and also added a list of Google services that are using GWT, that I knew of. I didn't remove other projects.

okrasz commented:
I've signed the CLA.

Best regards,
Marcin Okraszewski


On Mon, Mar 2, 2015 at 11:18 PM, googlebot <notifications@github.com> wrote:

> Thanks for your pull request.
>
> It looks like this may be your first contribution to a Google open source
> project, in which case you'll need to sign a Contributor License Agreement
> (CLA) at https://cla.developers.google.com/.
>
> If you've already signed a CLA, it's possible we don't have your GitHub
> username or you're using a different email address. Check the information
> on your CLA <https://cla.developers.google.com/clas> or see this help
> article on setting the email on your git commits
> <https://help.github.com/articles/setting-your-email-in-git/>.
>
> Once you've done that, please reply here to let us know. If you signed the
> CLA as a corporation, please let us know the company's name.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gwtproject/gwt-site/pull/57#issuecomment-76838899>.
>

jDramaix commented on src/main/markdown/examples.md:
```diff
@@ -144,13 +144,51 @@ GWT is being used by tens of thousands of projects around the world. Take a look
+    <div class="name"><a href="https://ruxit.com" target="_blank">ruxit</a></div>
+    <div class="snippet">ruxit is a cloud-based application performance management solution that offers unified monitoring of 
+user experience, servers, applications, infrastructure and networks &ndash; all with simple auto-configuration. The entire product UI 
+was built using GWT. Use the free trial to see it in action.</div>
```
*free trial*
If you have a direct link, add it

okrasz replied:
Unfortunately not. It seems there is an inline form on the main page only.

jDramaix commented on src/main/markdown/examples.md:
```diff
@@ -144,13 +144,51 @@ GWT is being used by tens of thousands of projects around the world. Take a look
+		<a href="https://www.google.com/producer" target="_blank">Currents&nbsp;Producer</a>,
+		<a href="http://www.google.com/flights" target="_blank">Flights</a>,
+		<a href="https://www.google.com/fusiontables/DataSource?dsrcid=2049253" target="_blank">Fusion&nbsp;Tables</a>,
+		<a href="http://www.google.com/hotelfinder" target="_blank">Hotel&nbsp;Finder</a>
```
@dankurka Could you review this part concerning Google ?

jDramaix commented:
Thanks for this contribution

tbroyer commented on src/main/markdown/examples.md:
```diff
@@ -144,13 +144,51 @@ GWT is being used by tens of thousands of projects around the world. Take a look
+		<a href="http://adwords.google.com " target="_blank">AdWords</a>,
+		<a href="https://www.google.com/webmasters/tools/home" target="_blank">Webmaster Tools</a>,
+		<a href="http://www.google.com/moderator/" target="_blank">Moderator</a>,
+		<a href="http://www.google.com/blogger" target="_blank">Blooger&nbsp;Admin</a>,
```
Typo: Blogger, not Blooger

tbroyer commented on src/main/markdown/examples.md:
```diff
@@ -144,13 +144,51 @@ GWT is being used by tens of thousands of projects around the world. Take a look
+	It is the first Google service that is using the new approach to delivering multiplatform native applications using common code base. GWT is used to deliver the Web version.
+	<p>Google is using GWT in number of other public services, including:
+		<a href="https://docs.google.com/spreadsheets" target="_blank">Spreadsheets</a>,
+		<a href="http://www.google.com/groups" target="_blank">Groups</a>,
```
https://groups.google.com/forum/ (or https://groups.google.com/ ), but please use HTTPS

tbroyer commented on src/main/markdown/examples.md:
```diff
@@ -144,13 +144,51 @@ GWT is being used by tens of thousands of projects around the world. Take a look
+	<p>Google is using GWT in number of other public services, including:
+		<a href="https://docs.google.com/spreadsheets" target="_blank">Spreadsheets</a>,
+		<a href="http://www.google.com/groups" target="_blank">Groups</a>,
+		<a href="http://adwords.google.com " target="_blank">AdWords</a>,
```
HTTPS (should be true for all links below too)

okrasz commented:
I've changed links to use HTTPS and fixed Blogger typo as @tbroyer suggested.
I've also improved a bit on the Inbox description, to reflect more the actual state (Spreadsheets was the first "hybrid" application) + added the link to Ray talking on this approach.

jDramaix commented:
Thanks !
